Lil-Peep/src/styles/modules/site-header.scss

```diff
@@ -16,8 +16,18 @@
 
     header {
       display: flex;
+      margin-bottom: 21px;
+      margin-top: 21px;
 
       .header-logo {
+        display: flex;
+        justify-content: center;
+        margin-bottom: 0;
```

Comment:
You could use shorthand property `margin: 0` for no margins on any side or `margin: 0 Xpx;` for no margins in top and bottom and X margin size in left and right.